packages/paste-codemods/tools/.cache/mappings.json

```diff
@@ -7,10 +7,6 @@
   "AccountSwitcherSeparator": "@twilio-paste/core/account-switcher",
   "useAccountSwitcherState": "@twilio-paste/core/account-switcher",
   "Alert": "@twilio-paste/core/alert",
-  "AlertBackgroundColors": "@twilio-paste/core/alert",
```

Comment:
issue: all these removals are technically breaking changes. Not sure how we want to handle it, but if someone downstream was using these it would break their app on upgrade

Reply:
<img width="945" alt="image" src="https://github.com/twilio-labs/paste/assets/75342690/264e62b8-4df4-4cdf-883d-3e049b4d6128">

I asked @SiTaggart the same thing on a previous PR, he's fine with removing anything that's not either:

1. documented
2. used elsewhere in our system

and not marking it as a breaking change.

Reply:
I'm easily convinced either way. 

My hesitation / second thoughts is do I want to deal with unintentional breakage. They shouldn't be using it, but if they are they don't care.

Reply:
I do kinda love how this check helps us see if we did a major booboo though on exports, super handy

Reply:
Ok just wanted to mention it. I think it's commonly understood in industry that undocumented things may break without warning.